crates/ruma-events/src/room/message/audio.rs

```diff
@@ -132,3 +140,11 @@ pub struct UnstableAudioDetailsContentBlock {
 #[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
Could you also derive `Default`?

Reply:
Applied!

Reply:
Wait, why? For the other type it makes sense, but I'm skeptical here. What good is it to initialize the details with a duration of zero?

Reply:
My bad, I put that on the wrong type.